spec/declaration.dd

```diff
@@ -9,6 +9,7 @@ $(GNAME Declaration):
     $(GLINK2 function, FuncDeclaration)
     $(GLINK VarDeclarations)
     $(GLINK AliasDeclaration)
+    $(GLINK AliasAssign)
```

Comment:
This isn't right because then it is syntactically ambiguous in function scope. It is the same syntax as `AssignExpression` (unless `Type` happens to be a `FundamentalType` or otherwise unambiguously a type).